### DISCUSSION TITLE
2sxc 17.3.0 locking up the UI on Oqtane 5.0.2-Latest

### DISCUSSION THREAD

**Discussion by maxvamp12** [2sic/2sxc#3323] [Q&A]
Hi all!! 

 I am absolutely new to Oqtane and 2sxc, but a very long time dot net developer. With that said, I want to add the disclaimer that the problem could still be between the keyboard and chair. 
 
  I have pulled the latest oqtane from master, and opened in VS2022. I built, ran ( in debug) , and configured for a remote SQL Server. All is happy with Oqtane. 
  
  I downloaded the install Nuget 2sxc 17.3.0 package and went through the UI in oqtane to install.  I then add the Content module to a page and I see 2sxc and its icon. I then hit the breadcrumb for the config ( the gear ) , and this action locks up all responsiveness from the app. 
  
   I am not sure what I am doing wrong. I am running in the IDE, using IIS Express, against a remote MS SQL DB  running in a docker container in a linux environment.  

Any help would be appreciated. Once I understand Oqtane and 2sxc better, I want to also start contributing. I would love to see this run with PostgreSQL, so that would be some work I would be happy to tackle.

**Comment by iJungleboy** [comment]
hi @maxvamp12 

You probably ran into a common problem in the dev setup.

Did you check out https://docs.2sxc.org/abyss/platforms/oqtane/install-issue-hot-reload.html ?

I did think that you should get a warning about this, but maybe something isn't working with our warning? 

Would be glad to get feedback if this fixed the issue.
